fenix/app/src/main/java/org/mozilla/fenix/compose/LinkText.kt

```diff
@@ -149,3 +156,24 @@ private fun LinkTextStyledPreview() {
         }
     }
 }
+
+@Preview
+@Composable
+private fun LinkTextClickStyledPreview() {
```

Comment:
Another open thought and discussion, but no action needed here. I am wondering if we should be continuing to grow our list previews like this. The alternative I am thinking is to have a single preview and have a column of all these `LinkText` variations. We could even inline the `LinkTextState`. 

*Pseudo code:*
```
val text = "This is clickable text, with underlined text"

Column(verticalAlignment = spacedBetween) { 
        LinkText(
                text = text,
                linkTextState = LinkTextState(...),
                style = FirefoxTheme.typography.headline5,
                linkTextColor = FirefoxTheme.colors.textOnColorSecondary,
                linkTextDecoration = TextDecoration.Underline,
            )
      
      LinkText(
                text = text,
                linkTextState = LinkTextState(...),
      )

      etc...
} 
```

I don't believe this is a problem right now, but if we continue to add more parameters and various previews, we might want to start making this a bit more concise otherwise we have more repetitive code displaying the previews than the actual implementation. Not necessarily a bad thing.

This comment is more so to capture thoughts out in the open and foster any input when other folks on the team revisit this PR. I think it's totally actionable if you feel strongly about one way or another. wdyt?

Reply:
referenced this in the follow-up ticket.